src/components/AddFundsInterstitial.js

```diff
@@ -115,9 +115,8 @@ const AmountButtonWrapper = styled(Row).attrs({
   ${android ? 'width: 100' : ''};
 `;
 
-const onAddFromFaucet = network => {
-  const faucetUrl = get(networkInfo[network], 'faucet_url');
-  Linking.openURL(faucetUrl);
+const onAddFromFaucet = () => {
+  Linking.openURL('https://faucet.paradigm.xyz');
```

Comment:
looks like line 275 is still passing along the `network`
```<InterstitialButton onPress={() => onAddFromFaucet(network)}```

also, you are defining a function inside of that onPress, should be able to just pass `onAddFromFaucet` now 

also you can remove the `{}` like:
```
const onAddFromFaucet = () => Linking.openURL('https://faucet.paradigm.xyz');
```

Reply:
lol oop

Reply:
fixed